frontend/apps/crates/components/src/input/simple_select/mod.rs

```diff
@@ -3,3 +3,18 @@ mod state;
 
 pub use dom::*;
 pub use state::*;
+
+pub trait Item: Clone {
+    fn value(&self) -> &str;
+    fn label(&self) -> &str;
+}
+
+impl Item for &str {
```

Comment:
This is useful for using lists of &str such as https://github.com/johnnynotsolucky/ji-cloud/blob/sandbox/frontend/apps/crates/entry/user/src/register/pages/step_2/dom.rs#L57

Reply:
:+1: